test_optimizer.py

```
@@ -1577,7 +1577,7 @@ def test_fitness_log1p_transform_behavior(self, mock_cfg, basic_metrics, empty_r
         metrics_positive_raw["sortino"] = 2.5
         
         _, _, diag_pos = optimizer._fitness_from_metrics(metrics_positive_raw, empty_rebal_log, mock_cfg)
-        expected_raw_pos = (100.0 / ((5.0 + (0.01 * 100.0 * 2.0) + 1.0) * 1.0)) * 1.0 - 0.0 - 0.0
+        _expected_raw_pos = (100.0 / ((5.0 + (0.01 * 100.0 * 2.0) + 1.0) * 1.0)) * 1.0 - 0.0 - 0.0
```

Flake8 (7.3.0)

[error] local variable '_expected_raw_pos' is assigned to but never used

(F841)